adapter/PreciceInterface.c

```diff
@@ -515,20 +522,27 @@ void PreciceInterface_ConfigureCouplingData( PreciceInterface * interface, Simul
 			interface->kDeltaWriteDataID = precicec_getDataID( config->writeDataNames[i], interface->faceCentersMeshID );
 			printf( "Write data '%s' found.\n", config->writeDataNames[i] );
 		}
-                else if ( strcmp1( config->writeDataNames[i], "Displacements" + i ) == 0 )
-                {
-                        PreciceInterface_EnsureValidNodesMeshID( interface );
-            		interface->writeData = DISPLACEMENTS;
-                        interface->displacementsDataID = precicec_getDataID( config->writeDataNames[i], interface->nodesMeshID );
-                        printf( "Write data '%s' found.\n", config->writeDataNames[i] );
-                }
+        else if ( strcmp1( config->writeDataNames[i], "Displacements" + i ) == 0 )
```

Comment:
White spaces seem to have been mixed up a little. Please correct this (at several positions in the code).